solutions/logging/core-experimentation/README.md

```diff
@@ -49,46 +48,55 @@ This package has no sub-packages.
 
 ## Usage
 
-1.  Clone the package:
+1. Clone the package:
+
     ```shell
     kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/logging/core-experimentation@${VERSION}
     ```
+
     Replace `${VERSION}` with the desired repo branch or tag
     (for example, `main`).
 
-1.  Move into the local package:
+1. Move into the local package:
+
     ```shell
     cd ".//solutions/logging/core-experimentation/"
     ```
 
-1.  Edit the function config file(s):
+1. Edit the function config file(s):
     - setters.yaml
 
-1.  Execute the function pipeline
+1. Execute the function pipeline
+
     ```shell
     kpt fn render
     ```
 
-1.  Initialize the resource inventory
+1. Initialize the resource inventory
+
     ```shell
     kpt live init --namespace ${NAMESPACE}
     ```
+
     Replace `${NAMESPACE}` with the namespace in which to manage
     the inventory ResourceGroup (for example, `config-control`).
 
-1.  Apply the package resources to your cluster
+1. Apply the package resources to your cluster
```

Comment:
same minor issue - numbered headings are not href referenceable - but this can be adjusted later if required - good as is

Reply:
this is a generated section from the kpt generate-doc utility, we cannot customize content between markers..marking as resolved